internal/tool/generate/generator_test.go

```diff
@@ -219,7 +225,7 @@ func TestGenerator(t *testing.T) {
 			}
 
 			// Run "weaver generate".
-			output, err := runGenerator(t, dir, filename, contents, []string{"sub1", "sub2"})
+			output, err := runGenerator(t, dir, filename, contents, []string{"sub1", "sub2"}, []string{})
```

Comment:
Can you just pass nil for the empty slice?

Reply:
Done